### DISCUSSION TITLE
UX e UI ainda não é levado muito a sério?

### DISCUSSION THREAD

**Discussion by omarkdev** [frontendbr/forum#479] [Discussão]
Recentemente cheguei ao seguinte pensamento e queria a opinião de vocês:

Os termos e funções de UX e UI estão cada vez mais comum na área de web e apps (ainda bem), porém não vejo muitas empresas ou desenvolvedores de softwares se preocupando com isso, é muito comum você baixar um software no seu computador e se estressar com ele (na maioria dos casos), devido a interface péssima ou muito mal desenvolvido. Porque vocês acham que nessa área não é comum? Algum dia será?

**Comment by kvnol** [comment]
Eu vejo que no Brasil a cultura do UX (todos têm responsabilidades) e do designer focado em UI ainda está crescendo no Brasil, diferentemente comparando com as empresas de fora que contratam diversos designers, etc. Muitas empresas, aqui do famoso _BRASA™_ não querem (até porque é caro, mas é um investimento)/entendem a importância de segmentar os grupos, com cada um na sua função sendo que podem contratar duas pessoas e cobrir a necessidade.

Por isso nós (desenvolvedores e gerentes atualizados com o mercado) temos que brigar nas empresa, mostrar dados e convencer o dono/supervisor (whatever) que UX e UI é sim importante, gerará vendas e deixará todos (TODOS) felizes.

**Todos temos responsabilidades com a UX**
Sim, muitas vezes vemos a sigla UX ligada apenas aos designers mas esquecemos que todos dentro da empresa (desde o faxineiro até o designer, do atendimento ao desenvolvedor) somos responsáveis pela experiência do nosso usuário (leia-se cliente) e precisamos desamarrar isso cada vez mais dos designers e amarrarmos a nós mesmo (seja qualquer função que você ocupe na empresa).

**Comment by bernardodiasc** [comment]
Pra aliviar um pouco a barra, não é uma questão de empresa de dentro ou de fora do Brasil, é questão de empresa boa ou ruim, mercado aquecido ou frio.

O problema que vejo nessa questão é gente que não sabe direito como as coisas funcionam no cargo de chefia. Pra mim isso resume todo o problema.

**Comment by Cleysonlb** [comment]
@kvnol concordo que todos na empresa tem responsabilidades ligadas a experiencia do usuario, mas ter um profissional de UX é indispensável, ele será o responsável direto que conhecerá os usuários, fará testes de usabilidade, irá tirar métricas entre outras coisas!

**Comment by kvnol** [comment]
@cleysonlb concordo 100% contigo. Eu não desconsiderei ter um profissional de UX dentro da empresa.

**Comment by talitapagani** [comment]
A UX e UI são deixadas em segundo plano em algumas empresas por pensar que isso tem a ver somente funções estéticas ou então pelo pensamento errôneo que isso não tem nada a ver com negócios. Ledo engano.

Eu costumo dizer que a interface é o "mensageiro" entre o seu código e as pessoas que utilizam o site/app. Se o mensageiro entrega a mensagem de forma errada ou indecifrável, a pessoa não vai conseguir entender ou o sistema não vai conseguir processar o que a pessoa quer. A UI dita como as informações são mostradas e a UX guia a pessoa na completude dos objetivos e visa atingir o bem-estar das pessoas. Desastres em qualquer um destes pontos podem gerar: queda de vendas (olha aí como tem a ver com negócio), diminuição de leads, estresse (como já comentado), queda de reputação da marca e até o famoso processinho™.

Atualmente, temos tantas técnicas para pesquisas com usuários que não convence mais o argumento de alguns gestores de que é caro e demora muito. Mesmo porque, se você investe em um processo de design correto deste o começo, você investe tempo em algo justo e economiza e dinheiro tempo em refações. E mesmo tendo um profissional de UX na empresa, que é responsável por orquestrar o processo de design, TODOS têm suas responsabilidades para construir um bom design de interação.

Esses dias vi uma frase do designer [Jared Spool](https://twitter.com/jmspool/status/836955987860914176) falando que "_qualquer pessoa que influencia no que o design (projeto) se torna é um designer e isso inclui desenvolvedores, gerentes de projetos e até a equipe legal_".

**Comment by rubenmarcus** [comment]
Sim e por isso dizem que os profissionais focados em UX e UI são caros e raros.. pra mim essa área parece bem nebulosa, pois vejo diversos perfis de profissionais, desde os que conseguem resolver o problema de comportamento de um usuário, e reverter lucro pra empresa ou o que quer que seja, até profissionais que eram webdesigners e se tornaram UX e mais UI designers por consequencia. Também tem profissionais que são ótimos UX/UI  e ótimos desenvolvedores.

Quanto ao dia-dia e a remuneração de cada um, já não sei.

Sei que to fazendo um curso no Coursera e não é bem simples quanto parece. Para muito chefe, pode parecer que um UX apenas desenha uns wireframes no papel, e depois faz um layout no Photoshop e Illustrator e é isso.

Eu mesmo queria saber como entrar nesse mercado e ir mais pra esse lado, porque gosto muito de Front, JS e etc, mas UX/UI pra mim é muito mais natural, mas as vagas são bem poucas, desde freelas pra trampo mesmo,  e a unica experiência que eu tenho é com trampos proprios meus..

**Comment by talitapagani** [comment]
@rubenmarcus puxa, eu sempre vejo muitas vagas. Sigo um grupo chamado UX Jobs no Facebook e alguns perfis no Twitter onde sempre há demandas. Como muito profissionais de UX acabam conhecendo o Github de alguma forma, será que repositório de vagas como o do frontendbr não seria interessante?

**Comment by rubenmarcus** [comment]
@talitapagani 

Bom eu não ando seguindo então, porque eu talvez veja mais as vagas de front. Mas vejo não só aqui,lá fora, parece que tem mais vagas pra front e back do que pra UX ou Webdesign.

Essa separação ainda esta nebulosa pra mim porque UX =/ UI  e UI parece muito parecido com Webdesign.


O que eu tenho que fazer pra conseguir um trampo de UX ? alem de preencher os requisitos? To fazendo um curso do Coursera aqui https://www.coursera.org/specializations/interaction-design

Sei que ai está apenas o começo. Um segundo passo seria refazer meu portfolio. E depois? Porque experiencia mesmo eu só tenho com webdesign ( criação de logos,layouts,campanhas etc.) Mas quero ir pra UX/UI

PS: Pra mim meu trampo de Front-end está legal. Eu quero pegar freelas de UX/UI pra criar expertise mesmo. Trocaria de trampo só se fosse uma proposta muito legal mesmo..

**Comment by talitapagani** [comment]
@rubenmarcus para quem está migrando de área, como foi o meu caso e é o seu, realmente é meio difícil, mas não impossível. Portfólio é algo que conta muito e não precisa ser algo muito extenso. Seus trabalhos na área de Webdesign podem ser úteis. Para um portfólio legal de UX, é interessante descrever o processo que você levou para produzir o site: pesquisou concorrentes? a que conclusões chegou? entrevistou usuários? como foram os primeiros wireframes? como foi a discussão com o time para ajustar? houve algum teste com protótipos? Essas são algumas questões que colocamos em cada projeto de UX, o mais importante é entender o seu raciocínio e seu processo. UX é um termo guarda-chuva onde está inserido a UI e envolve muito o processo para solucionar um problema de negócio. Espero ter ajudado!

**Comment by talitapagani** [comment]
Ah, e se você não tiver muitos trabalhos, pegue algum site existente e proponha um redesign da interface e do fluxo de interação, documentando isso. É uma boa forma de demonstrar também suas skills.

**Comment by rubenmarcus** [comment]
ótimo!! tem o behance de uma UX Designer aqui que só tem redesigns de sites famosos, Mas são redesigns muito bons ! e vejo templates free de Mobile Apps também. Acho que é um bom começo !!

**Comment by rubenmarcus** [comment]
https://www.behance.net/search?content=projects&sort=appreciations&time=week&search=UX

Referências

**Comment by Cleysonlb** [comment]
@talitapagani fazer um processo de UX para por no portfólio é ótimo! Estou nesse processo, sou front end e aos poucos estou migrando, estou criando um produto "fake" que levará todos os processos de pesquisa, teste, prototipação, wireframe e design, pretendo documentar tudo e usar como referencia!!!

**Comment by talitapagani** [comment]
@Cleysonlb show! É esse o caminho!

**Comment by rubenmarcus** [comment]
Alguem já usou esse software da Adobe ? 

http://www.adobe.com/br/products/experience-design.html


Usam qual no dia-dia? Balsamiq?

**Comment by talitapagani** [comment]
Como eu sou usuária de Windows (sim, rs), muitos softwares interessantes como o Adobe XD (que só agora tem pra Windows 10) e o Sketch não estão disponíveis, então recorro a algumas ferramentas online. Atualmente tenho usado muito o [Moqups.com](http://moqups.com) e, olha, é excelente para construir desde protótipos rápidos até layouts em alta fidelidade. Tem suporte a hotspots para fazer mockups interativos e bibliotecas de componentes de UI do Bootstrap, Material Design e outros, recomendo muito.

**Comment by Cleysonlb** [comment]
@rubenmarcus o Adobe XD é muito bom, perfeito para fazer teste de usabilidade de forma rápida e facil, mas ainda continuo usando o Sketch, hoje o Sketch consegue facilmente substituir um photoshop/illustrator na criação de layouts (interfaces)!

**Comment by rubenmarcus** [comment]
É eu uso Windows tambem.. 
bem..
e cursos e livros?

Estava vendo alguns cursos aqui:

https://webdesign.tutsplus.com/courses/becoming-a-ux-developer?ec_promo=learning_roundup
https://www.interaction-design.org/

alguem recomenda algum?

**Comment by bernardodiasc** [comment]
"No budget? No excuse. Here’s a practical guide to UX on the cheap." https://medium.freecodecamp.com/no-budget-no-excuse-heres-a-practical-guide-to-ux-on-the-cheap-9605c77ef55b

**Comment by WandersonAlves** [comment]
Terminei de ler a issue e muito obrigado @talitapagani. Pretendo até a metade do ano fazer uma especialização em Iteração, mais precisamente essa: http://www.cesar.edu.br/course/especializacao-em-design-de-interacao-para-artefatos-digitais-dipad/

Será que vale a pena fazer isso?

Quanto as ferramentas, eu costumava brincar numa artboard do Photoshop, prototipando em alta fidelidade e jogando as prints no http://marvelapp.com ou no http://invisionapp.com (particularmente acho o Marvel melhor). Também andei usando o http://frontify.com pra guardar o styleguide e de quebra, a documentação da aplicação que ando fazendo.

O problema é que, como usuário windows, a artboard do Photoshop não me é muito intuitiva. Pretendo testar essa versão beta do Adobe XD e ver se consigo melhorar o fluxo.

De pouco em pouco a gente vai crescendo hehe.

E o engraçado que lá em 2010, quando estava decidindo fazer uma faculdade, pensei em fazer faculdade de Design e não a de Ciência da Computação. Hoje, me formado em CC, pretendo voltar a área de Design porém de uma forma mais especifica;

**Comment by tarciozemel** [comment]
Se em "Algum dia será"? "Lá fora" já é: o pessoal se preocupa bem mais; o nível de profissionalismo e de entregas é superior. Na Terra dos Papagaios, o [pessoal mal sabe a diferença/responsabilidade entre ambos](http://desenvolvimentoparaweb.com/design/ux-ui-ia-ixd/).

Regra geral, por aqui, desde pequenas produtoras a grandes agências, "fazer UX" é adaptar um template ou copiar um site na cara dura alterando o que julgam necessário para o projeto. E os "designers" ainda exigem ser chamados de "criativos"...

**Comment by talitapagani** [comment]
Creio que vale a pena fazer uma pós na área, @WandersonAlves. Tem também a [pós da Universidade Positivo](http://www.up.edu.br/pos-graduacao/design-centrado-no-usuario-design-de-interacao), que tem um bom corpo docente.

Sei que as ferramentas são importantes, mas não se prenda a elas. Na prática, posso te dizer que trabalhar com UX e design de interação tem muito mais a ver com o processo do que com as ferramentas.

Ah, eu também quase fiz faculdade de Design, depois optei por Ciência da Computação e agora estou no mesmo caminho inverso de voltar para o Design.

**Comment by WandersonAlves** [comment]
@talitapagani Que bom que funcionou contigo, já é um gás pra mim.

Viva T.I e sua versatilidade!

**Comment by headofnoone** [comment]
Pessoal,

estou fazendo uma pesquisa sobre esse assunto. Qual a opinião de vocês sobre o mercado de UX no Brasil?

É só responder esse link: https://freshstarts.typeform.com/to/d8o3IY

Os resultados serão divulgados e a gente espera que ajudem a fortalecer o mercado de UX.

Mais sobre nós: https://www.facebook.com/somosnoone/
Obrigada!
